.github/workflows/workflow.yml

```diff
@@ -393,7 +392,7 @@ jobs:
         # Restore kubeconfig from remote docker host.
```

Comment:
should this comment say that we're getting the kubeconfig from kind?

Reply:
From what I understand, `kind export kubeconfig` retrieves the config from the remote host, so the comment is accurate